mmdet3d/datasets/utils.py

```diff
@@ -1,21 +1,59 @@
+# yapf: disable
 from mmdet3d.datasets.pipelines import (Collect3D, DefaultFormatBundle3D,
                                         LoadAnnotations3D,
+                                        LoadImageFromFileMono3D,
                                         LoadMultiViewImageFromFiles,
                                         LoadPointsFromFile,
-                                        LoadPointsFromMultiSweeps)
+                                        LoadPointsFromMultiSweeps,
+                                        MultiScaleFlipAug3D,
+                                        PointSegClassMapping)
+# yapf: enable
 from mmdet.datasets.builder import PIPELINES
 from mmdet.datasets.pipelines import LoadImageFromFile
 
 
+def is_loading_function(transform):
+    """Judge whether a transform function is a loading function.
+
+    Args:
+        transform (dict | :obj:`Pipeline`): A transform config or a function.
+
+    Returns:
+        bool | None: Whether it is a loading function. None means can't judge.
+            When transform is `MultiScaleFlipAug3D`, we return None.
+    """
+    # TODO: use more elegant way to distinguish loading modules
+    loading_functions = (LoadImageFromFile, LoadPointsFromFile,
+                         LoadAnnotations3D, LoadMultiViewImageFromFiles,
+                         LoadPointsFromMultiSweeps, DefaultFormatBundle3D,
+                         Collect3D, LoadImageFromFileMono3D,
+                         PointSegClassMapping)
```

Comment:
A little dangerous. We may not remember to add new defined loading functions to here when implementing a new one. Is there any other good way? Or at least we need to move them into __init__ or somewhere else such that it is convenient for us to remember to update them when adding new functions?

Reply:
Do you think it's ok if we regard all functions defined in `formating.py` and `loading.py` as loading functions? I have checked and this rule applies to all functions except one (`NormalizePointsColor`, which I do think should be placed in `transform_3d.py` instead of `loading.py`). What do you think @ZwwWayne?

Reply:
I asked Wenwei and he said currently we can just leave it as is. Maybe make it better in the future.